### PR TITLE
Update container versions of NumPy and TensorFlow.

### PR DESCRIPTION
**Pull request by robertwb**
These are expected by TFT and TFMA. This saves a significant amount of worker startup time when running TFX pipelines on portability. 

The versions in the container are also quite old by now. 

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @mxm

File: sdks/python/container/Dockerfile [hunk 1]
```diff
@@ -73,11 +73,11 @@ RUN \
     pip install "guppy == 0.1.10" && \
     pip install "python-snappy == 0.5.3" && \
     # These are additional packages likely to be used by customers.
-    pip install "numpy == 1.13.3" --no-binary=:all: && \
+    pip install "numpy == 1.15" && \
     pip install "pandas == 0.18.1" && \
     pip install "scipy == 1.0.0" && \
     pip install "protobuf == 3.3.0" && \
-    pip install "tensorflow == 1.4.0" && \
+    pip install "tensorflow == 1.9" && \
```
Comment (mxm) [inline sdks/python/container/Dockerfile]: Do we support upgrading the provided packages? What if user code requires tensorflow 1.4?

**Comment by robertwb** [comment]
Yes, we support specifying packages to be installed at runtime, but that
takes time. Also, tensorflow (attempts to) use semantic versioning, so a
user should be requiring tensorflow >=1.x,<2.0.

**Comment by robertwb** [comment]
Jenkins: retest this please

**Pull request merged**